qemu/tests/rng_hotplug.py

```diff
@@ -37,26 +37,16 @@ def get_rng_id(vm):
 
     def hotplug_rng(vm, dev):
         error_context.context("Hotplug %s" % dev, logging.info)
-        output = dev.hotplug(vm.monitor)
-        time.sleep(5)
-
-        error_context.context("Check %s from qtree after hotplug" % dev,
-                              logging.info)
-        qtree_output = dev.verify_hotplug(output, vm.monitor)
-        if not qtree_output:
+        out, ver_out = vm.devices.simple_hotplug(dev, vm.monitor)
+        if not ver_out:
             msg = "no % device in qtree after hotplug" % dev
             raise exceptions.TestFail(msg)
```

Comment:
Better to emit `output` in error message.

Reply:
Why not use `test.fail()`?

Reply:
Okay ignore me, that's nothing to do with this pull request.